apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -38,6 +58,87 @@ export default function LearningGoal({
     setIsOpen(!isOpen);
   };
 
+  const handleFeedbackChange = event => {
+    if (studentLevelInfo.user_id && learningGoal.id) {
+      if (autosaveTimer.current) {
+        clearTimeout(autosaveTimer.current);
+      }
+      teacherFeedback.current = event.target.value;
+      setDisplayFeedback(teacherFeedback.current);
```

Comment:
The state is used to keep the textarea updated as the user types, and the ref is used for autosave, because state doesn't update until after re-render, and that was causing the autosaved info to lag behind what was actually typed. I couldn't think of a good way to combine these.